### ISSUE TITLE
Enhancement Request - Move Multiple Effects Vertical

### ISSUE THREAD

**Issue by Phrog30** [smeighan/xLights#726]
Currently there is a way to move multiple effects horizontally, but I don't see anyway to do this vertically.  This is not important, just something to save a little time here and there.  Thank you for considering.

James

**Comment by AzGilrock** [comment]
Did you try the Up and Down arrows?  It should move them as long as the space is clear.

**Comment by Phrog30** [comment]
Yes, the arrows will move a single effect, but I found no way to move multiple.  If it's possible, then please let me know, I just haven't seen a way.

**Comment by AzGilrock** [comment]
The arrows will move multiple effects.  You gotta select them first.  I tried it out before I replied.

**Comment by Phrog30** [comment]
I tried it before submitting in the first place.  But, I just tried it 
again.  I selected two effects by dragging a box around them.  I then 
hit the left arrow, they moved left.  I hit the right arrow, they moved 
right.  I hit the up arrow, nothing.  I hit the down arrow, nothing.  
So, you are saying that you are able to move MULTIPLE effects UP and 
DOWN?  I can move a single effect up and down, just not multiple.  I 
would make a video of this, but there would be nothing to see, and I 
have described it very clearly.

James

------ Original Message ------
